src/services/dashboards/widgets/type.ts

```diff
@@ -16,6 +18,6 @@ export interface XYChartData {
 
 export interface Legend {
     name: string;
-    label?: string;
+    label?: TranslateResult;
```

Comment:
todo: `label` of `Legend` is not `TranslateResult`. It's reference data. (ex. Belkin Snow)